Library/Koala/Koala.swift

```diff
@@ -1295,6 +1325,20 @@ public final class Koala {
     self.track(event: "Facebook Friend Unfollow", properties: ["source": source.trackingString])
   }
 
+  // MARK: Update Events
+
+  public func trackViewedUpdate(forProject project: Project, context: UpdateContext) {
+    let props = properties(project: project, loggedInUser: self.loggedInUser)
+      .withAllValuesFrom(["context": context.trackingString])
+
+    self.track(event: "Viewed Update", properties: props)
+  }
+
+  public func trackViewedUpdates(forProject project: Project) {
+    let props = properties(project: project, loggedInUser: self.loggedInUser)
+    self.track(event: "Viewed Updates", properties: props)
+  }
```

Comment:
oh wait... these are tracked on the web side, we don't need to track them in the app.

let's chat irl with @ktman